obj.go

```diff
@@ -420,6 +448,33 @@ func (O ObjectCollection) Trim(n int) error {
 	return O.drv.checkExec(func() C.int { return C.dpiObject_trim(O.dpiObject, C.uint32_t(n)) })
 }
 
+func (O ObjectCollection) CollAsArrayMap() ([]map[string]interface{}, error) {
```

Comment:
What if the collection's elements are not objects?

Reply:
The godror.ObjectCollection could be called even with native types?
I create the collection starting from type, so i suppose it's always a custom type.